dockerfiles/amdgpu_ubuntu_ghascale.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ghcr.io/actions/actions-runner:latest
+
+RUN sudo apt-get update -y \
+    && sudo apt-get install -y software-properties-common \
+    && sudo add-apt-repository -y ppa:git-core/ppa \
+    && sudo apt-get update -y \
+    && sudo apt-get install -y --no-install-recommends \
+    curl \
+    ca-certificates \
+    git \
+    jq \
+    sudo \
+    unzip \
+    zip \
+    cmake \
+    ninja-build \
+    clang \
+    lld \
+    wget \
+    psmisc \
+    python3.10-venv \
```

Comment:
Let's merge this first with no changes and try migrating the runners to use it, but after migration we could do some cleanup.

One nitpick I'd have would be to merge the `apt-get install` and `apt install` commands if possible and move `python3.10-venv` next to `libpython3.10`.